gptme/tools/save.py

```diff
@@ -116,26 +146,58 @@ def execute_append(
         return
 
     with open(path, "a") as f:
-        f.write(code)
+        f.write(content)
     yield Message("system", f"Appended to {fn}")
 
 
 tool_save = ToolSpec(
     name="save",
     desc="Write text to file",
     instructions=instructions,
+    instructions_format=instructions_format,
     examples=examples,
     execute=execute_save,
     block_types=["save"],
+    parameters=[
+        Parameter(
+            name="filepath",
```

Comment:
I prefer the shorter `path` (many occurrences)

```suggestion
            name="path",
```

Reply:
Fixed.